.generator/templates/model_utils.mustache

```diff
@@ -1419,15 +1419,12 @@ def get_oneof_instance(cls, model_kwargs, constant_kwargs, model_arg=None):
                         constant_kwargs['_check_type'],
                         configuration=constant_kwargs['_configuration']
                     )
-            oneof_instances.append(oneof_instance)
+            if not oneof_instance._unparsed:
+	        oneof_instances.append(oneof_instance)
         except Exception:
             pass
     if len(oneof_instances) == 0:
-        raise ApiValueError(
-            "Invalid inputs given to generate an instance of %s. None "
-            "of the oneOf schemas matched the input data." %
-            cls.__name__
-        )
+        return UnparsedObject(**model_kwargs)
     elif len(oneof_instances) > 1:
```

Comment:
Should the same be done for more than 1 instance matching ?

Reply:
Good question, I'm not sure. I don't know if we had scenarios with this in the past? OTOH it looks like what you did in Go, so we should probably align.